pkg/fsutil/fsutil_unix.go

```diff
@@ -0,0 +1,11 @@
+//go:build !windows && !plan9
+// +build !windows,!plan9
+
+package fsutil
+
+import "os"
+
+// IsExecutable returns true if the stat object refers to an executable file on UNIX platforms.
```

Comment:
When exporting a function with platform-dependent implementations, I'd prefer putting the entry point in a shared file, even if it is just a call to the unexported version. This makes it the Godoc easier to read.